### PR TITLE
Jeremie/add task

### PR DESCRIPTION
**Pull request by jeremielim**

### CODE REVIEW

File: now/now/AddTaskViewController.swift [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        taskNameTextField.becomeFirstResponder()
+    }
+    
+    @IBAction func didTapCancel(sender: AnyObject) {
+        // Close modal
+        dismissViewControllerAnimated(true, completion: nil)
+    }
+    
+    @IBAction func didTapSave(sender: AnyObject) {
+        let curTaskCount = NSUserDefaults.standardUserDefaults().integerForKey("taskCount")
```
Comment (jeremielim) [inline now/now/AddTaskViewController.swift]: Remove this line

File: now/now/AddTaskViewController.swift [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        var newTask =  Task(withTaskName: taskName!, categoryName: categoryName!, andTime: time!)
+        
+        tasks.append(newTask)
+        
+        let data = NSKeyedArchiver.archivedDataWithRootObject(tasks)
+        
+        // Add object to NS user defaults
+        NSUserDefaults.standardUserDefaults().setObject(data, forKey: defaultsKey)
+        
+        NSUserDefaults.standardUserDefaults().setInteger(curTaskCount + 1, forKey: "taskCount")
```
Comment (jeremielim) [inline now/now/AddTaskViewController.swift]: Remove this line

File: now/now/TaskListViewController.swift [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        
+        tableView.delegate = self
+        tableView.dataSource = self
+        
+        NSUserDefaults.standardUserDefaults().setInteger(-1, forKey: "taskCount")
+    }
+    
+    override func viewWillAppear(animated: Bool) {
+        let curTaskCount = NSUserDefaults.standardUserDefaults().integerForKey("taskCount")
+
```
Comment (jeremiespoken) [inline now/now/TaskListViewController.swift]: Add comment here

File: now/now/TaskListViewController.swift [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        super.viewDidLoad()
+        
+        tableView.delegate = self
+        tableView.dataSource = self
+        
+        NSUserDefaults.standardUserDefaults().setInteger(-1, forKey: "taskCount")
+    }
+    
+    override func viewWillAppear(animated: Bool) {
+        let curTaskCount = NSUserDefaults.standardUserDefaults().integerForKey("taskCount")
```
Comment (jeremielim) [inline now/now/TaskListViewController.swift:29]: Remove task count variable

File: now/now/TaskListViewController.swift [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var clearView: UIView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.delegate = self
+        tableView.dataSource = self
+        
+        NSUserDefaults.standardUserDefaults().setInteger(-1, forKey: "taskCount")
```
Comment (jeremielim) [inline now/now/TaskListViewController.swift]: Delete this

File: now/now/TaskListViewController.swift [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        return tasks.count
+    }
+    
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        
+        let cell = tableView.dequeueReusableCellWithIdentifier("TaskCell")! as! TaskTableViewCell
+        
+        cell.taskLabel?.text = tasks[indexPath.row].taskName
+        cell.taskCategory?.text = tasks[indexPath.row].categoryName
+        cell.taskTime?.text = "\(tasks[indexPath.row].time):00"
```
Comment (jeremielim) [inline now/now/TaskListViewController.swift:59]: Find time format, store in seconds and convert to minutes

**Pull request merged**